xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-api/src/main/java/org/xwiki/refactoring/internal/ModelBridge.java

```diff
@@ -64,6 +64,15 @@
      */
     boolean delete(DocumentReference documentReference);
 
+    /**
+     * Delete the specified document while explictly skiping the recyclebin, even if it exists and is activated.
+     *
+     * @param documentReference the reference of the document to delete
+     * @return {@code true} if the document was deleted successfully, {@code false} if the delete failed
+     * @since 12.8RC1
+     */
+    boolean expurge(DocumentReference documentReference);
```

Comment:
Why not `delete(DocumentReference documentReference, boolean toRecycleBin)` and have `  boolean delete(DocumentReference documentReference)` call it?

Reply:
`delete(DocumentReference documentReference, boolean toRecycleBin)` introduced (see https://github.com/xwiki/xwiki-platform/pull/1369/files#diff-0665e5b7ff47f7f52c23f66192734100R75).